dev-tools/getArticlesToCheckSyndicationStatus.js

```diff
@@ -0,0 +1,68 @@
+// This dev tool uses a raw list of syndication downloads and outputs a list of who syndicated an article by a freelancer and when. For more information, see the readme. 
+
+/* eslint-disable guard-for-in */ // removeDuplicates is from StackOverflow and this function is run manually, we can apply this rule to the whole file
+/* eslint-disable no-console */ // console logs are there to track progress
+const esClient = require('@financial-times/n-es-client');
+const fs = require('fs');
+const { downloads } = require('../syndicationDownloads');
+
+async function getContent(articles) {
+
+	// data we want to get from Elastic Search for each article
+	const elements = ['id', 'canBeSyndicated', 'title', 'byline'];
+
+	// map over the articles to create array to pass to esClient
+	const docsToQuery = articles.map(article => {
+		return { _id: article.uuid, _source: elements };
+	});
+
+	// console.log('================')
+	// console.log("docsToQuery.length", docsToQuery.length);
+	// console.log('================')
+
+	// if working with HUGE files, slice the array to have length ~15000 each, esClient chokes on larger arrays. If your file is smaller, you may need to amend the code so it doesn't slice and concat and runs `const someData = await esClient.mget({ docs: docsToQuery }, 30000);` and then `const notYes = someData.filter(item => item.canBeSyndicated === 'withContributorPayment');`
+	const sliced1 = docsToQuery.slice(0, 15100);
```

Comment:
**Suggestion**: Can we give our variables more meaningful names than `sliced1` or `someData1`?